action.yml

```diff
@@ -33,6 +33,9 @@ inputs:
     description: "List of target platforms for build (e.g. linux/amd64,linux/arm64,linux/riscv64,linux/ppc64le,linux/s390x,etc)"
     required: false
     default: 'linux/amd64'
+  provenance:
+    description: "Generate provenance attestation for the build"
+    required: false
```

Comment:
The problem is that the default changes based on the buildx version, but I can add one

Reply:
See: https://github.com/docker/build-push-action/blob/master/src/context.ts#L137-L152

Reply:
Adding a default is literarily a breaking change in this case I think because public repos will get either `mode=min,inline-only=true`. I also see that `build-args` is not required and has no default